<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QStackedWidget" name="stackedWidgetVistas">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page1CnpMetOrto">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QGroupBox" name="groupBoxCoordinatesFile">
            <property name="title">
             <string>Coordinates file</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="TableViewCoordinates" name="widgetCoordinates" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBoxOptionsCnpMetOrto">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="title">
                <string>Options Cnp Met Orto</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="IndexButton" name="pushButtonOptionsCnp">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="text">
                       <string>Options cnp</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="IndexButton" name="pushButtonOptionsMet">
                      <property name="text">
                       <string>Options met</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="IndexButton" name="pushButtonOptionsOrto">
                      <property name="text">
                       <string>options orto</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QStackedWidget" name="stackedWidgetOpciones">
                    <widget class="CreateCnps" name="page1Cnp"/>
                    <widget class="CreateMet" name="page2Met"/>
                    <widget class="CreateOrto" name="page3Orto"/>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonEmpezarOperacionesMetOrto">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBoxTextEditCnpMetOrto">
           <property name="title">
            <string>Incidens Cnp Met Orto</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTextEdit" name="textEditCnpMetOrto">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2renameChangeFormats">
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Select sensor</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="IndexButton" name="pushButtonCasi">
                   <property name="text">
                    <string>CASI</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_2">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="IndexButton" name="pushButtonTasi">
                   <property name="text">
                    <string>TASI</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_3">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="IndexButton" name="pushButtonAisa">
                   <property name="text">
                    <string>AISA</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidgetCasitasiAisa">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="TableViewRaw" name="page1ChangeFormatsCasi"/>
              <widget class="TableViewRawTasi" name="page2ChangeFormatsTasi"/>
              <widget class="QWidget" name="page3ChangeFormatsAisa"/>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBoxTextEditChangeFormats">
           <property name="title">
            <string>Incidents Change formats</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTextEdit" name="textEditChangeFormats"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3CombineCnp">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="CombineCnp" name="widgetCombineCnp" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBoxTextEditCombineCnp">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QTextEdit" name="textEditCombineCnp"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4Geocorrection">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="Correction" name="widget" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBoxTextEditCorrection">
           <property name="title">
            <string>Incidents Correction</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QTextEdit" name="textEditCorrection">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page5DiferenceOrtoCorrectio">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="Diferences" name="widget_2" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBoxTextEditCombinaCnp_2">
           <property name="title">
            <string>Incidents differences Orto Pass</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QTextEdit" name="textEditCombinaCnp_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page6ViewInformHtml"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="actionPreferencias"/>
   </widget>
   <widget class="QMenu" name="menu_Help_2">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_Qt"/>
    <addaction name="actionAbout_ToolsPcot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Help_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSave_project"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
	background-color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCnps_met_orto"/>
   <addaction name="actionCnps_rename_change_format"/>
   <addaction name="actionCnps_combine_cnps"/>
   <addaction name="actionCnps_correction"/>
   <addaction name="actionCnps_diferences"/>
   <addaction name="actionCnps_view_html"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/document-new.png</normaloff>:/imagenes/imagenes/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/document-open.png</normaloff>:/imagenes/imagenes/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/document-save.png</normaloff>:/imagenes/imagenes/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save project</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/document-save-as.png</normaloff>:/imagenes/imagenes/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;ave as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/application-exit.png</normaloff>:/imagenes/imagenes/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionAbout_ToolsPcot">
   <property name="text">
    <string>About &amp;ToolsPcot</string>
   </property>
  </action>
  <action name="actionCnps_met_orto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/cnp_met_orto.png</normaloff>:/imagenes/imagenes/cnp_met_orto.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCnps_rename_change_format">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/rename_change_formats.png</normaloff>:/imagenes/imagenes/rename_change_formats.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCnps_combine_cnps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/combine_cnps.png</normaloff>:/imagenes/imagenes/combine_cnps.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCnps_correction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/geo-correction.png</normaloff>:/imagenes/imagenes/geo-correction.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCnps_diferences">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/Diference_orto_correctio.png</normaloff>:/imagenes/imagenes/Diference_orto_correctio.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCnps_view_html">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/imagenes/view_inform_HTML.png</normaloff>:/imagenes/imagenes/view_inform_HTML.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionPreferencias">
   <property name="text">
    <string>&amp;Preferencias</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableViewCoordinates</class>
   <extends>QWidget</extends>
   <header>tableviewcoordinates.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateCnps</class>
   <extends>QWidget</extends>
   <header>createcnps.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateMet</class>
   <extends>QWidget</extends>
   <header>createmet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateOrto</class>
   <extends>QWidget</extends>
   <header>createorto.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableViewRaw</class>
   <extends>QWidget</extends>
   <header>tableviewraw.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IndexButton</class>
   <extends>QPushButton</extends>
   <header>indexbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TableViewRawTasi</class>
   <extends>QWidget</extends>
   <header>tableviewrawtasi.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CombineCnp</class>
   <extends>QWidget</extends>
   <header location="global">combinecnp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Correction</class>
   <extends>QWidget</extends>
   <header location="global">correction.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Diferences</class>
   <extends>QWidget</extends>
   <header location="global">diferences.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
